ecl/hqlcpp/hqlttcpp.cpp

```diff
@@ -13709,6 +13804,13 @@ void HqlCppTranslator::normalizeGraphForGeneration(HqlExprArray & exprs, HqlQuer
     allocateSequenceNumbers(exprs);                                             // Added to all expressions/output statements etc.
 
     traceExpressions("allocate Sequence", exprs);
+
+    if (options.transformNestedSequential)
+    {
+        transformNestedSequential(exprs);
+        traceExpressions("allocate Sequence", exprs);
```

Comment:
traceExpressions label incorrect ("allocate Sequence") - label should be renamed